autoload/ddc_file/internal.vim

```diff
@@ -2,8 +2,8 @@ function! s:line_to_file_full(line, is_posix) abort
   return matchlist(
     \ a:line,
     \ a:is_posix
-       \ ? '[[:fname:]/]*$'
-       \ : '[[:fname:]\\]*$'
+       \ ? '\c\%(^\$\%(env:\)\?\)\?[[:fname:]/]*$'
+       \ : '\c\%(^\$\%(env:\)\?\)\?[[:fname:]\\]*$'
```

Comment:
You may think this should be done by vim users themselves, but if we edit .ps1 file in linux, fname does not include colon by default, and detecting colon **only in head** does not affect so destructively.